### ISSUE TITLE
Not being able to switch back from DTD schema editor

### ISSUE THREAD

**Issue by iztokkosem** [elexis-eu/lexonomy#164]
If I insert my own DTD configuration and then at some point want to go back to the default Lexonomy schema editor, it is not accepted. Once I turn DTD option off, I get shown the Lexonomy framework-like schema, but I cannot save it and DTD is still being used. 
If I delete the DTD content and then switch it off (stop using my own schema), I cannot even insert any data in the default schema editor.
Basically, whenever I do something and come back to the settings, the DTD is still being used.

I'm assigning this to various people - once someone can take this over, you can remove the others.
